pr2eus_tutorials/euslisp/pr2-tabletop-object-grasp.l

```diff
@@ -210,15 +210,15 @@
   (setq *grasp-state* "returning")
   (publish-grasp-state)
   (pr2-pregrasp-pose)
-  (publish-info "click tabletop object in Rviz!")
-  (publish-info "click tabletop object in image_view2!")
+  (ros::ros-info "click tabletop object in Rviz or in image_view2!")
```

Comment:
```
(defun publish-info (&rest args)
  (let ((s (format nil (car args) (cdr args))))
    (ros::ros-info s)
    (publish-status 'info s)))
```
`publish-info` calls both `ros::ros-info` and 'publish-status`, so why you need to call `ros::ros-info` twice?

Reply:
You're right. (I forgot why I don't use `publish-info`)

Reply:
This is because 
```
(ros::ros-info  (format nil "click tabletop object~%in Rviz or in image_view2!"))
```
called by 
```
(publish-info (format nil "click tabletop object~%in Rviz or in image_view2!"))
```
splits the sentence on the command line, which is slightly difficult to read.
![Screenshot from 2021-11-17 17-14-10](https://user-images.githubusercontent.com/42209144/142161271-16990b2b-c0eb-4194-8e74-eb8c0b389d30.png)